stable/sonarqube/README.md

```diff
@@ -102,6 +102,7 @@ The following table lists the configurable parameters of the Sonarqube chart and
 | `plugins.initSysctlContainerImage`          | Change init sysctl container image        | `busybox:1.31`                             |
 | `plugins.deleteDefaultPlugins`              | Remove default plugins and use plugins.install list | `[]`                             |
 | `podLabels`                                 | Map of labels to add to the pods          | `{}`                                       |
+| `runShFolder`                               | Directory name of the run.sh script       | `sonarqube`                                |
```

Comment:
I suggest to rename it to `sonarqubeFolder`. The default should be `/opt/sonarqube'. This way it works even if sonarqube is moved outside of `/opt*/